jetty-proxy/src/test/java/org/eclipse/jetty/proxy/ForwardProxyTLSServerTest.java

```diff
@@ -908,6 +907,7 @@ public void handle(String target, Request baseRequest, HttpServletRequest reques
 
     @ParameterizedTest
     @MethodSource("proxyTLS")
+    @DisabledIfSystemProperty(named = "env", matches = "ci")
```

Comment:
I reproduce this locally for every run

Reply:
Don't disable it, let's try to find out why it fails.
Also this cannot be reproduced locally by me, so it's either Jenkins/Docker/K8s.